src/main/features/claimant-response/views/confirmation.njk

```diff
@@ -42,13 +42,10 @@
         <p>
           <span class="bold">{{ confirmationDate | date }}</span>
         </p>
-        {% if directionsQuestionnaireEnabled %}
```

Comment:
Please put this back in. It is unlikely we will turn it off but it should still remain wrapped in the feature toggle.